app-misc/mdate/mdate-1.7.0.3.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="A converter to Mayan dates"
+DESCRIPTION="Mdate is a utility that converts Gregorian dates to Mayan Long Count dates. \
+It was originally an implementation of some algorithms published by N. Dershowitz and \
+E.M. Reingold in their book Calendrical Calculations."
+HOMEPAGE="https://github.com/ewe2/mdate"
+COPYRIGHT="1989 - 2014 Ty Coon"
```

Comment:
Please fix the COPYRIGHT:
```
-COPYRIGHT="1989 - 2014 Ty Coon"
+COPYRIGHT="2010-2014 Sean Dwyer"
```
(Ty Coon is the author of the GPL license, not of the program.)